docs/provisioner/08-02-provisioning-gardener.md

```diff
@@ -67,6 +67,22 @@ This tutorial shows how to provision clusters with Kyma Runtimes on Google Cloud
   > **NOTE:** To get the AWS IAM policy, access your project on Gardener, navigate to the **Secrets** tab, click on the help icon on the AWS card, and copy the JSON policy. 
     
   </details>
+  
+  <details>
+    <summary label="OpenStack">
+    OpenStack
+    </summary>
+    
+   - Existing project on Gardener
+   - OpenStack project with technical user available to create, modify and delete VMs
```

Comment:
```suggestion
   - OpenStack project with technical user available to create, modify, and delete Virtual Machines (VMs)
```

Are there any specific roles that need to be assigned to the user or specific credentials needed? Like the ones we mention for other providers. This would help the user to set it up right away instead of googling how to let this technical user do this.